bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat

```diff
@@ -0,0 +1,281 @@
+ 0   = ht3min ! minimum Ht for the three leading jets
+ 0   = ht4min ! minimum Ht for the four leading jets
+ -1  = ht2max ! maximum Ht for the two leading jets
+ -1  = ht3max ! maximum Ht for the three leading jets
+ -1  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+   0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ .true. = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0   = xetamin ! minimum rapidity for two jets in the WBF case  
+ 0   = deltaeta ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1    =  ktdurham        
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+   T = use_syst
+#
+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
+#**************************************                                  
+#
+0.5 1 2 = sys_scalefact  # factorization/renormalization scale factor
```

Comment:
@perrozzi @kdlong @qliphy @agrohsje  what do we do with these lines?
https://github.com/anirban1982/genproductions/blob/20399ef110f1243311178be5b9c4a2b3a0ffb0b1/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat#L276-L281

Reply:
@perrozzi @kdlong @qliphy @agrohsje @efeyazgan good point indeed. I saw in

https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

says

SysCalc is not used anymore and these lines should be removed: 

Should I remove the line 276 to 281?

https://github.com/anirban1982/genproductions/blob/20399ef110f1243311178be5b9c4a2b3a0ffb0b1/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat#L276-L281

Reply:
They are irrelevant, so probably best to remove them. Syscalc is used but is configured independently in the gridpack_generation scripts.